src/renderer/components/kube-object-menu/dependencies/kube-object-menu-items/kube-object-menu-registration.d.ts

```diff
@@ -4,7 +4,6 @@
  */
 
 import type React from "react";
-import { BaseRegistry } from "./base-registry";
 
 export interface KubeObjectMenuComponents {
   MenuItem: React.ComponentType<any>;
```

Comment:
This would probably the best time to actually fix this with the correct prop types here. Which are:

```typescript
export interface KubeObjectMenuItemProps {
  object: KubeObject;
  toolbar?: boolean;
}
```